src/extensions/surveys.tsx

```diff
@@ -356,16 +363,20 @@ export function Questions({
         originalQuestionIndex: number
         displayQuestionIndex: number
     }) => {
-        const isLastDisplayedQuestion = displayQuestionIndex === survey.questions.length - 1
+        if (!posthog) {
```

Comment:
when does this happen?

Reply:
`posthog` is an optional argument to `SurveyPopup` (we're not passing it in the preview mode), so this needs to be checked